test/integration/scheduler/scheduler_test.go

```diff
@@ -179,33 +177,38 @@ func TestSchedulerCreationFromConfigMap(t *testing.T) {
 		eventBroadcaster.StartRecordingToSink(&clientv1core.EventSinkImpl{Interface: clientSet.CoreV1().Events("")})
 
 		defaultBindTimeout := int64(30)
-		ss := &schedulerappconfig.Config{
-			ComponentConfig: kubeschedulerconfig.KubeSchedulerConfiguration{
-				HardPodAffinitySymmetricWeight: v1.DefaultHardPodAffinitySymmetricWeight,
-				SchedulerName:                  v1.DefaultSchedulerName,
-				AlgorithmSource: kubeschedulerconfig.SchedulerAlgorithmSource{
-					Policy: &kubeschedulerconfig.SchedulerPolicySource{
-						ConfigMap: &kubeschedulerconfig.SchedulerPolicyConfigMapSource{
-							Namespace: policyConfigMap.Namespace,
-							Name:      policyConfigMap.Name,
-						},
+
+		sched, err := scheduler.New(clientSet,
```

Comment:
we need this call in extender_test.go as well

Reply:
extender_test.go uses initTestScheduler, which in turn finally calls scheduler.NewFromConfig() in test/integration/scheduler/util.go

Reply:
start here https://github.com/kubernetes/kubernetes/blob/master/test/integration/scheduler/extender_test.go#L352

Reply:
Ah, that's a good point. Looks like we'll need a follow-up PR to get `test/integration/util` onto the new constructor.